spec.emu

```diff
@@ -9,6 +9,135 @@ contributors: Google, Ecma International
+      This proposal re-adds the section to ECMA-402, but with additional requirements for Intl-supporting implementations, beyond those specified in ECMA-262.
+    </p>
+  </emu-note>
+
+  <del class="block">
+    <p>This specification identifies calendars using a <dfn variants="calendar types">calendar type</dfn> as defined by <a href="https://unicode.org/reports/tr35/tr35-dates.html#Calendar_Elements">Unicode Technical Standard #35 Part 4 Dates, Section 2 Calendar Elements</a>. Their canonical form is a string containing only Unicode Basic Latin lowercase letters (U+0061 LATIN SMALL LETTER A through U+007A LATIN SMALL LETTER Z) with zero or more medial hyphens (U+002D HYPHEN-MINUS).</p>
+  </del>
+
+  <ins class="block">
+    <p>
+      ECMA-262 describes calendar types, of which *"iso8601"* is required to be supported.
+      This specification additionally requires ECMAScript implementations to support calendar types corresponding with those of the <a href="https://cldr.unicode.org/">Unicode Common Locale Data Repository (CLDR)</a>.
+    </p>
+  </ins>
+
+  <emu-clause id="sup-availablecalendars" oldids="sec-availablecalendars sec-availablecanonicalcalendars" type="implementation-defined abstract operation">
+    <h1>AvailableCalendars ( ): a List of calendar types</h1>
+    <dl class="header">
+      <dt>description</dt>
+      <dd>The returned List is sorted according to lexicographic code unit order, and contains unique calendar types in canonical form (<emu-xref href="#sec-calendar-types"></emu-xref>) identifying the calendars for which the implementation provides the functionality of Intl.DateTimeFormat objects, including their aliases (e.g., <del>either</del> both <del>or neither of</del> *"islamicc"* and *"islamic-civil"*). The List must include <del>*"iso8601"*</del><ins>the Calendar Type value of every row of <emu-xref href="#table-calendar-types"></emu-xref>, except the header row</ins>.</dd>
+    </dl>
+    <p><ins>This definition supersedes the definition provided in <emu-xref href="#sec-availablecalendars"></emu-xref>.</ins></p>
+  </emu-clause>
+
+  <ins class="block">
+    <emu-table id="table-calendar-types">
+      <emu-caption>Calendar types described in CLDR</emu-caption>
+      <table class="real-table">
+        <thead>
+          <tr>
+            <th>Calendar Type</th>
+            <th>Description and implementation notes</th>
+          </tr>
+        </thead>
+        <tr>
+          <td>*"buddhist"*</td>
+          <td>Thai Buddhist calendar, proleptic. Month numbers, month codes, and days are the same as in the ISO 8601 calendar, but the epoch year is different. There is one era.</td>
+        </tr>
+        <tr>
+          <td>*"chinese"*</td>
+          <td>Traditional Chinese calendar, proleptic. Similar lunisolar algorithm to *"dangi"*. The arithmetic year is identical to *"gregory"* and there are no eras.</td>
```

Comment:
We probably want to talk about the fallback of Chinese and Dangi, too. The issue is in `proposal-temporal` instead of `proposal-intl-era-monthcode`. https://github.com/tc39/proposal-temporal/issues/2869

But we can land this now and fix it up later.